src/assisted_test_infra/test_infra/helper_classes/cluster.py

```diff
@@ -1380,9 +1380,7 @@ def format_host_from_config_mapping_file(config: str) -> dict[str, dict[str, Lis
                         for address_version in consts.IP_VERSIONS.keys():
                             if address_version not in current_interface.keys():
                                 continue
-                            address = current_interface[address_version]["address"]
-                            if len(address) == 0:
-                                continue
```

Comment:
this is made for empty IP address in interface
make sure you test all possible deployment static IPv6 / bonding / static IPv4

Reply:
Yes , we have configuration when the ip is empty , like static configuration with dhcp

Reply:
the bonding and IPv6 should also run on this PR - just to make sure
but, as far as I see `lgtm`

Reply:
yes , in kni-test-infra we have bond test and it works with this change